tests/py_localhost/server.py

```diff
@@ -81,7 +86,12 @@ def data_received(self, data: bytes):
     def request_received(self, headers: List[Tuple[str, str]], stream_id: int):
         self.raw_headers = headers
         headers = collections.OrderedDict(headers)
+        path = headers[':path']
         method = headers[':method']
+        if method == "PUT" or method == "POST":
+            self.file_path = os.path.join(os.path.curdir, path[1:])
+            if os.path.exists(self.file_path):
+                os.remove(self.file_path)
```

Comment:
and this is why we're not running a public webserver.
An attacker could pass a path like `/../../../../some_important_program` and replace files with whatever they want

But this isn't public, so we can do whatever hacky crap we want